composer.json

```diff
@@ -21,7 +21,16 @@
 		"php:compatibility": "composer install && vendor/bin/phpcs -p -s --runtime-set testVersion '5.3-' --standard=PHPCompatibilityWP --ignore=docker,tools,tests,node_modules,vendor --extensions=php",
 		"php:lint": "composer install && vendor/bin/phpcs -p -s",
 		"php:autofix": "composer install && vendor/bin/phpcbf",
-		"php:lint:errors": "composer install && vendor/bin/phpcs -p -s --runtime-set ignore_warnings_on_exit 1"
+		"php:lint:errors": "composer install && vendor/bin/phpcs -p -s --runtime-set ignore_warnings_on_exit 1",
+		"modifyPackageTextDomain": [
+			"./node_modules/node-wp-i18n/bin/wpi18n addtextdomain --textdomain=jetpack --glob-pattern='(packages|vendor)/**/*.php'"
```

Comment:
I wonder if we should call `npm install` before running this one, just to make sure this will work regardless?

Reply:
I'm really just wondering about what we actually do in production. I was told `npm install` is run first, but given how things are run in CI, I'm not too sure myself.

Reply:
Doing the same thing as a manual `yarn` task is absolutely fine by me for instance.

Reply:
Well, I'm talking about the case where none of them have ran yet. In that case an `npm install` (or in our case the equivalent is `yarn install-if-deps-outdated`) before that would assure that we have the wp18n before running it, regardless of when we're running it.